evento/src/views/EventPage/index.js

```diff
@@ -36,20 +54,46 @@ class EventPage extends Component {
 		});
 	}
 
+	onAttendButtonClick() {
+		const eventId = this.props.match.params.eventId;
+
+		fetch(`/events/${eventId}/attendees`, {
+			method: 'POST',
+			headers: { 'Authorization': Cookie.get('auth_token') }
+		})
+		.then(response => {
+			if (!response.ok) return Promise.reject();
+			// If successiful attending, re-fetch attendees
+			this.fetchAttendees();
+		})
+		.catch(() => this.setState({ errorMessage: "Something went wrong" }));
+	}
+
+	isUserAttending() {
+		let user = Cookie.get('user');
+		return JSON.stringify(this.state.attendees).includes(user);
```

Comment:
Well `user` is stored as json so it would need to be parsed before checking so it would look something like this:
```js
let user = Cookie.get('user');
if (user !== undefined) {
    user = JSON.parse(user);
}
return this.state.attendees.some(attendee => attendee.id == user.id);
```

I prefer the existing one but `let` should be `const`.